Flutter/installation/Installation-errors.md

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: Installation Errors in Flutter - Syncfusion
 description: Learn here about the common installation errors and solution to those errors in Syncfusion Flutter Components installation.
```

Comment:
Hi @AswiniDileep,

Check and ensure once with @LavanyaGowtham2021 Mam that we dont need to update trademark symbol here.

Regards,
Hari Hara Sudhan. K.

Reply:
Okay.